package.json

```diff
@@ -44,6 +44,7 @@
     "ts-helpers": "^1.1.1"
   },
   "dependencies": {
+    "ajv": "^4.11.3",
```

Comment:
@jmartinm I checked that this is a warning other than error and is related with webpack. There are some proposed solutions but we must change webpack config and now is not provided from angular-cli for configuring webpack config from package.json like karma. Here is a link with the critical-dependencies issue in webpack:
https://github.com/benjamine/jsondiffpatch/issues/76